malaria24/ona/tests/test_tasks.py

```diff
@@ -18,6 +19,10 @@
 
 from .base import MalariaTestCase
 
+FAKE_URL = "http://jembi.org/malaria24"
```

Comment:
If you move these to settings, you can make this the default values for when the env vars aren't present. Then you wouldn't need to override the settings.

Reply:
Noted. Thanks